pkg/server/endpoints/node/handler.go

```diff
@@ -624,3 +570,114 @@ func getSpiffeIDFromCSR(csr []byte) (spiffeID string, err error) {
 
 	return spiffeID, nil
 }
+
+func FetchRegistrationEntries(ctx context.Context,
```

Comment:
Maybe this code would be better placed in a dedicated file? `entries.go`, `fetch_entries.go`, or something similar?

Reply:
good idea. i intended to move it into its own package (could be useful to cmdline utilities or other packages outside of the node handler). I'll move it.